pkg/integrations/windows_exporter/config_windows.go

```diff
@@ -0,0 +1,226 @@
+package windows_exporter //nolint:golint
```

Comment:
Can you rebase? I think this is going to fail lints since it's missing godocs for some exported methods, and I don't want the lint to fail after the merge.